packages/core/src/entities/BlockTool.ts

```diff
@@ -0,0 +1,27 @@
+import type { BlockToolAdapter } from '@editorjs/dom-adapters';
+import type { BlockTool as BlockToolVersion2 } from '@editorjs/editorjs';
+import type { BlockToolConstructorOptions as BlockToolConstructorOptionsVersion2 } from '@editorjs/editorjs';
+
+/**
+ * Extended BlockToolConstructorOptions interface for version 3.
+ */
+export interface BlockToolConstructorOptions extends BlockToolConstructorOptionsVersion2 {
+  /**
+   * Block tool adapter will be passed to the tool to connect data with the DOM
+   */
+  blockToolAdapter: BlockToolAdapter;
```

Comment:
Maybe just `adapter` as this is a block tool by declaration

Reply:
done